src/domain/app/AppNonProfileRoute.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import AppPageTitleRoute from './AppPageTitleRoute';
+import { RouteProps } from 'react-router';
```

Comment:
Imports from `node_modules` should come before in-app imports based on the linting rules in this project as far as I know. Maybe your IDE didn't integrate with eslint successfully?